testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -191,8 +191,9 @@
   fill_in('command', with: cmd, fill_options: { clear: :backspace })
 end
 
-When(/^I enter target "([^"]*)"$/) do |minion|
-  fill_in('target', with: minion, fill_options: { clear: :backspace })
+When(/^I enter target "([^"]*)"$/) do |host|
+  value = $is_cloud_provider ? get_system_name(host) : SEARCH_VALUE[host]
```

Comment:
Personally I don't see the advantage of having the SEARCH_VALUE hash.
If what we want in the remote command scenario is to target the hostname of one of our hosts, I would say it can works fine just by doing:

```suggestion
  value = get_system_name(host)
```

In all cases :thinking: 

Otherwise, if we want to continue having the previous behavior, then just do:

```suggestion
  value = $is_cloud_provider ? get_system_name(host) : host
```

Not modifying the regex values passed by parameter in the steps.

But I would go with the first option.

Reply:
Hi, as discussed in Slack, my opinion about it, is that I think is fine if we don't cover the test of the wildcard feature in this field, if by losing this coverage we gain in transparency between VMs/Cloud and we gain in maintenability.